BadgeUpClient/Http/HttpQuery.cs

```diff
@@ -22,5 +25,27 @@ public override string ToString()
 
 			return null;
 		}
+
+
+		public static string GetQueryStringFromObject(object param)
```

Comment:
Wouldn't it make more sense to move this to an interface that all of the *QueryParams classes extend so we can override individual implementations if needed?

Reply:
This one is generic enough to convert any object to a query string, so I thought that HttpQuery is an OK place for it. Can be moved to base class for QueryParams if you prefer.

Reply:
Let's move it so we don't have issues where someone tries to serialize some random object